config/containers/logging/configure.md

```diff
@@ -43,32 +44,25 @@ example sets two configurable options on the `json-file` logging driver:
 {
   "log-driver": "json-file",
   "log-opts": {
-    "max-size": "10m",
-    "max-file": "3",
     "labels": "production_status",
     "env": "os,customer"
   }
 }
 ```
 
-> **Note**: `log-opt` configuration options in the `daemon.json` configuration
-> file must be provided as strings. Boolean and numeric values (such as the value
-> for `max-file` in the example above) must therefore be enclosed in quotes (`"`).
 
 If you do not specify a logging driver, the default is `json-file`. Thus,
 the default output for commands such as `docker inspect <CONTAINER>` is JSON.
 
 To find the current default logging driver for the Docker daemon, run
 `docker info` and search for `Logging Driver`. You can use the following
-command:
+command on Linux, macOS, or PowerShell on Windows:
 
-{% raw %}
 ```bash
-$ docker info --format '{{.LoggingDriver}}'
```

Comment:
I think this one was added so that it worked on any platform (not just linux)

Reply:
Added original info back in - please let me know if I misunderstood your comment.